src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import './Clock.scss';
+
+type Props = {
+  name: number;
+};
+
+type State = {
+  time: string;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    time: '',
+  };
+
+  timerId: NodeJS.Timer = setInterval(() => {}, 0);
+
+  componentDidMount() {
+    const intervalCb = () => {
```

Comment:
you can create this method on a component and use it (your componentDidMount will be cleaner)

```suggestion
    const updateTime = () => {
```